### PR TITLE
Introduce global CONTRIBUTING metadata inside the GitHub organization

### PR DESCRIPTION
**Pull request by oleg-nenashev**
CC @olblak @MarkEWaite

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Contributing to Jenkins Infrastructure
+======================================
+
+Our infrastructure is an open infrastructure project made by and for the Jenkins community.
+In other words, it’s a contributors driven project.
```
Comment (MarkEWaite) [inline CONTRIBUTING.md]:
```suggestion
In other words, it’s a contributor driven project. 
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+Contributing to Jenkins Infrastructure
+======================================
+
+Our infrastructure is an open infrastructure project made by and for the Jenkins community.
+In other words, it’s a contributors driven project. 
+While we can’t share publicly everything like secrets, we try to be as transparent as possible so everybody can read,
```
Comment (MarkEWaite) [inline CONTRIBUTING.md]:
```suggestion
While we can’t share everything publicly (like secrets), we try to be as transparent as possible so everybody can read, 
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+Contributing to Jenkins Infrastructure
+======================================
+
+Our infrastructure is an open infrastructure project made by and for the Jenkins community.
+In other words, it’s a contributors driven project. 
+While we can’t share publicly everything like secrets, we try to be as transparent as possible so everybody can read, 
+understand, improve our infrastructure without having privileged accesses.
```
Comment (MarkEWaite) [inline CONTRIBUTING.md]:
```suggestion
understand, and improve our infrastructure without having privileged accesses.
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (oleg-nenashev) [reply CONTRIBUTING.md]: Copy-paster from jenkins.io

**Pull request merged**